### PR TITLE
comment out block range for sepolia

### PR DESCRIPTION
**Pull request by jchoi330**

### CODE REVIEW

File: packages/frontend/src/context/StoreContext.tsx [hunk 1]
```diff
@@ -429,7 +436,10 @@ export const StoreContextProvider = (
 
   const addProductToTag = async (tagId: TagId, itemId: ItemId) => {
     try {
-      await relayClient!.updateTag({ tagId, addItemId: itemId });
+      await relayClient!.updateTag({
+        tagId: hexToBytes(tagId),
+        removeItemId: hexToBytes(itemId),
```
Comment (cryptix) [inline packages/frontend/src/context/StoreContext.tsx:441]: i think this needs to be `addItemId`

**Review by cryptix — CHANGES_REQUESTED** [review]
i think there is bug/wrong field name in one of the updateTag() calls but rest looks good!

File: packages/frontend/src/context/StoreContext.tsx (hunk 1 above)
Comment (jchoi330) [reply packages/frontend/src/context/StoreContext.tsx:441]: oh good catch

**Review by cryptix — APPROVED** [review]
👏🏻

**Pull request merged**